docs/sources/reference/commandline/cli.md

```diff
@@ -870,7 +877,7 @@ Creates a new container.
       --read-only=false          Mount the container's root filesystem as read only
       --restart="no"             Restart policy (no, on-failure[:max-retry], always)
       --security-opt=[]          Security options
-      -t, --tty=false            Allocate a pseudo-TTY
+      -t, --tty=false            Allocate a pseudo-TTY (NOTE: Do not use with -a stderr)
```

Comment:
I don't think this change should be made because this is the actual output from "usage"